echemdb/data/cv/database.py

```diff
@@ -194,3 +194,29 @@ def __repr__(self):
 
         """
         return repr(list(self))
+    
+    def get_entry(self, identifier):
+        r"""
+        Return an entry with the given identifier.
+
+        EXAMPLES::
+
+            >>> database = Database.create_example()
+            >>> database.get_entry('alves_2011_electrochemistry_6010_p2_f2a_solid')
+            Entry('alves_2011_electrochemistry_6010_p2_f2a_solid')
+
+        """
+        return [entry for entry in self if entry.identifier == identifier][0]
+    
+    def __getitem__(self, identifier):
+        r"""
+        Return an entry with the given identifier.
+
+        EXAMPLES::
+
+            >>> database = Database.create_example()
+            >>> database['alves_2011_electrochemistry_6010_p2_f2a_solid']
+            Entry('alves_2011_electrochemistry_6010_p2_f2a_solid')
+
+        """
+        return [entry for entry in self if entry.identifier == identifier][0]
```

Comment:
When there is no such entry, this will throw ` IndexError: list index out of range`. That's maybe confusing. I think you need to manually throw a `KeyError` when there is no such entry.

Also, when there is more than one such entry, then this will silently return the first one. While, this should not happen in a sane database, there's nothing that keeps a user from adding the same identifier twice. So we should maybe at least produce a warning when that happened.